src/caffegraphinput.cc

```diff
@@ -189,7 +189,6 @@ namespace dd
         parse_simple_lstm(net);
         return 0;
       }
-    return 0;
+    return -1;
```

Comment:
This fix is unrelated to this PR.

Reply:
this allows to throw smthg if proto do not contains a properly defined LSTM. Will change with smthg proper